### ISSUE TITLE
Chrome XDM extension Not Work (Emergency)

### ISSUE THREAD

**Issue by uniqueitmail** [subhra74/xdm#1168]
-----------------------------------------------------**_HELP*_*---------------------------

### **Getting error into chrome extension tab:::** **_(v8.0.29 BETA)_**

Uncaught (in promise) Error: Cannot access a chrome:// URL
Context
main.js
Stack Trace
main.js:0 (anonymous function)

![1001](https://github.com/subhra74/xdm/assets/16665892/8e2ddee1-68d5-4092-8a51-e1b3873c5443)
![1002](https://github.com/subhra74/xdm/assets/16665892/39cf8464-6f26-456c-8f6a-a0bd868f7b89)

**Comment by joselmm** [comment]
Hi there, i am develover, you can try my [option](https://github.com/subhra74/xdm/issues/1162) for now (i updated xdm 7.2.11, and also you can download youtube videos)

**Comment by 64daysofstarman** [comment]
In the case that you are not using a mac, you can bypass this. You have to download the beta version of xdm, go to the three bars, and select browser monitoring. Select chrome, and there it should give you the right extension.

**Comment by 64daysofstarman** [comment]
Don't download from any other source than the app.

**Comment by colibris79** [comment]
> In the case that you are not using a mac, you can bypass this. You have to download the beta version of xdm, go to the three bars, and select browser monitoring. Select chrome, and there it should give you the right extension.

It doesn't give me the right site. [The link](https://subhra74.github.io/xdm/redirect.html?target=chrome&version=8.0), just opens an error page from the chrome extension website

**Comment by 64daysofstarman** [comment]
Ok then it is stored on your computer. Try to find an address where its stored on your computer. I ran into the same error.

**Comment by 64daysofstarman** [comment]
It is stored at C:\Users\yourusername\xdm-app-data\chrome-extension assuming your using windows

**Comment by colibris79** [comment]
> It is stored at C:\Users\yourusername\xdm-app-data\chrome-extension assuming your using windows

Thanks for your suggestions. I am using Linux. The folder is /home/.xdm-app-data. Nothing there. I manually downloaded and installed from the [xdm-integration-module github](https://github.com/subhra74/xdm-integration-module)
It is installed but the media/video grabber does not find any links or media like the old version did

**Comment by mdsalmanmasud** [comment]
Dear XDM Developer's Team,

I am writing to you today as a valued user of Xtreme Download Manager (XDM). I am a big fan of the extension and its features, and I have been using it for many years.

However, I am concerned about the current version of XDM using Manifest Version 2. Unfortunately, Google has removed support for MV2 as of December 2023, meaning XDM will no longer function in future Chrome updates.

Using an outdated extension can pose security risks and hinder compatibility with upcoming features. Therefore, I strongly encourage you to consider migrating XDM to Manifest Version 3. This will ensure its continued functionality and user safety.

I understand that migrating to a new manifest version can be challenging. If there's anything I can do to help with the process, such as providing feedback or testing, please don't hesitate to ask.

Thank you for your time and consideration. I look forward to hearing from you and continuing to use XDM for many years to come.

Sincerely,

md.salmanmasud

**Comment by joselmm** [comment]
> Dear XDM Developer's Team,
> 
> I am writing to you today as a valued user of Xtreme Download Manager (XDM). I am a big fan of the extension and its features, and I have been using it for many years.
> 
> However, I am concerned about the current version of XDM using Manifest Version 2. Unfortunately, Google has removed support for MV2 as of December 2023, meaning XDM will no longer function in future Chrome updates.
> 
> Using an outdated extension can pose security risks and hinder compatibility with upcoming features. Therefore, I strongly encourage you to consider migrating XDM to Manifest Version 3. This will ensure its continued functionality and user safety.
> 
> I understand that migrating to a new manifest version can be challenging. If there's anything I can do to help with the process, such as providing feedback or testing, please don't hesitate to ask.
> 
> Thank you for your time and consideration. I look forward to hearing from you and continuing to use XDM for many years to come.
> 
> Sincerely,
> 
> md.salmanmasud

if you are referring to XDM 7.2.11 , i updated its extension to manifestv3 [here](https://github.com/subhra74/xdm/issues/1162)

**Comment by AkiyaKiko** [comment]
I solved it. In [8.0.29](https://github.com/subhra74/xdm/releases/tag/8.0.29), there is no app.js in the extension folder. Just go to [Repo](https://github.com/subhra74/xdm/blob/master/app/XDM/chrome-extension/app.js) and download the app.js and paste to your extension path.

**Comment by dephacto** [comment]
I installed 8.0.29 and added app.js to extension folder however the same error occurs
![image](https://github.com/user-attachments/assets/b9056a60-e533-4945-b3f8-a6a3f08f81b9)

**Comment by inuonly** [comment]
yup having same error . not working in any chrome based browser , tested in chrome , chromium, brave. 

> I installed 8.0.29 and added app.js to extension folder however the same error occurs ![image](https://private-user-images.githubusercontent.com/60345384/356095485-b9056a60-e533-4945-b3f8-a6a3f08f81b9.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjUxMjgzMjMsIm5iZiI6MTcyNTEyODAyMywicGF0aCI6Ii82MDM0NTM4NC8zNTYwOTU0ODUtYjkwNTZhNjAtZTUzMy00OTQ1LWIzZjgtYTZhM2YwOGY4MWI5LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA4MzElMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwODMxVDE4MTM0M1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTAyZDNiYTRhZDU2MDQ0NTI4MjA5MzA3MzE3ZDczZDlhOTRkYzQyNTAxZGNjOWMzYzAyZDUxOWFmYmNmYTE5NzkmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.wyfkSzE6vkvr3-DYCrr_eCKkFtClBml_wpdmpjfb4RM)
